docs/api/commands.rst

```diff
@@ -28,11 +28,29 @@ Command Types
 .. autoclass:: MinValue()
 .. autoclass:: ChannelTypes()
 
-
-
 ChatCommandHandler
-~~~~~~~~~~~~~~~~~~
+------------------
 
 .. attributetable:: ChatCommandHandler
 
 .. autoclass:: ChatCommandHandler()
+
+
+Message Components
+~~~~~~~~~~~~~~~~~~
+.. currentmodule:: pincer.commands.components
+
+.. autoclass:: ActionRow()
+.. autoclass:: Button()
+.. autoclass:: LinkButton()
+.. autoclass:: ButtonStyle()
```

Comment:
You forgor to put members lmao

Reply:
wdym

Reply:
> wdym

well like :member:

Reply:
Idk what you want. Its what I wanted it to look like.